src/store/metadata/stub/content/datasets.json

```diff
@@ -1,22 +1,28 @@
 {
-    "items": [
+    "@context": "https://staging.idpd.uk/#ns",
+    "@type": ["dcat:Catalog", "hydra:Collection"],
+    "@id": "http://localhost:8000/datasets",
+    "title": "Consumer Price Inflation including owner occupiers' housing costs (CPIH)",
+    "datasets": [
         {
-            "@context": "https://data.ons.gov.uk/ns#",
-            "@id": "https://data.ons.gov.uk/datasets/cpih",
+            "@id": "https://staging.idpd.uk/datasets/cpih",
             "@type": "dcat:DatasetSeries",
             "identifier": "cpih",
             "title": "Consumer Price Inflation including owner occupiers' housing costs (CPIH)",
             "summary": "The Consumer Prices Index including owner occupiers' housing costs (CPIH) is a measure of inflation that includes costs associated with...",
             "description": "The Consumer Prices Index including owner occupiers' housing costs (CPIH) is a measure of inflation that includes costs associated with...",
-            "release_date": "2017-01-01T00:00:00",
+            "issued": "2017-01-01T00:00:00",
+            "modified": "2017-01-01T00:00:00",
             "next_release": "2019-04-19T09:30:00Z",
-            "publisher": "office-for-national-statistics",
-            "creator": "office-for-national-statistics",
+            "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
```

Comment:
Our `@base` will be set for many of these fields, like [publisher,creator] only requiring the kebab case for the entities pointing to the URL for them on the gov.uk website.

So I'd expect to see `office-for-national-statitics` here. @rossbowen

Reply:
We had some discussion yesterday about themes and publishers, and whether they should be IRIs or kebab'd strings here. I'm still thinking kebab'd strings, but we agreed to do IRIs for now and do a small check with some suppliers to see what they favoured.